src/raven.js

```diff
@@ -1560,10 +1624,20 @@ Raven.prototype = {
     return globalServer;
   },
 
-  _handleOnErrorStackInfo: function() {
+  _handleOnErrorStackInfo: function(stackInfo, options) {
+    options = options || {};
```

Comment:
We can leave `this._handleStackInfo.apply(this, arguments);` by changing mechanism assignment to

```js
options = options || {};
options.mechanism = options.mechanism || {
  type: 'onerror',
  handled: false
}
```

which will also get rid of `objectMerge` call

Reply:
Changed the mechanism assignment as requested, but would rather not use `arguments` as this kills many potentials for optimizations in the JavaScript VM.